csrc/mlp.cpp

```diff
@@ -1,10 +1,39 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Should this be changed to 2021?

Reply:
At least add the -2021.  We can come back later and delete the 2020- part if it's inaccurate.